e2e/_suites/fleet/fleet.go

```diff
@@ -41,6 +41,7 @@ type FleetTestSuite struct {
 	Installers          map[string]installer.ElasticAgentInstaller
 	Integration         kibana.IntegrationPackage // the installed integration
 	Policy              kibana.Policy
+	FleetPolicy         kibana.Policy
```

Comment:
Good catch! At first place, the fleet config struct calculated the fleet policy on creation, and it was removed